ibm/service/power/data_source_ibm_pi_networks.go

```diff
@@ -0,0 +1,128 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func DataSourceIBMPINetworks() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMPINetworksRead,
+		Schema: map[string]*schema.Schema{
+			// Arguments
+			Arg_CloudInstanceID: {
+				Description:  "The GUID of the service instance associated with an account.",
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			// Attributes
+			Attr_Networks: {
+				Computed:    true,
+				Description: "List of all networks.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						Attr_AccessConfig: {
+							Computed:    true,
+							Description: "The network communication configuration option of the network (for satellite locations only).",
+							Type:        schema.TypeString,
+						},
+						Attr_DhcpManaged: {
+							Computed:    true,
+							Description: "Indicates if the network DHCP Managed.",
+							Type:        schema.TypeBool,
+						},
+						Attr_Href: {
+							Computed:    true,
+							Description: "The hyper link of a network.",
+							Type:        schema.TypeString,
+						},
+						Attr_Jumbo: {
```

Comment:
Since this `Attr_Jumbo` is deprecated, why are we showing it?

Reply:
Removed deprecated field.